Scripts/Runtime/Entities/TaskSystem/Job/Wrapper/GenericDataAccessWrapper.cs

```diff
@@ -0,0 +1,32 @@
+using Anvil.Unity.DOTS.Jobs;
+using Unity.Jobs;
+
+namespace Anvil.Unity.DOTS.Entities.Tasks
+{
+    internal class GenericDataAccessWrapper<TData> : AbstractAccessWrapper
```

Comment:
Should this just be `DataAccessWrapper`?
I think the only other subclass on `AbstractAccessWrapper` now is the CDFE one right?

Reply:
No we have a few:

![Screen Shot 2022-10-17 at 3 00 43 PM](https://user-images.githubusercontent.com/339008/196260384-e6518378-3874-4fa4-b735-595ee6cf7412.png)

I'm ok renaming to `DataAccessWrapper` since it's internal and the user won't ever see it, I just felt like it signified that it was this general case for when it doesn't fit into the more specialized flows.

Reply:
Nope, good rationale. Thanks!